lib/unit-ruby/account_limits.rb

```diff
@@ -0,0 +1,108 @@
+module Unit
+  class AccountLimits < APIResource
```

Comment:
Since there was quite a bit of overloading required to mack this quack correctly, what do you think about moving this into the `Accounts` API resource?

```Ruby
class DepositAccount < APIResource
  def limits
    @cached_limits ||= connection.get("#{resource_path(id)}/limits")
    Types::Limits.cast(cached_limits)
  end
  
  def ach_debit_daily_limit
    limits.dig('ach', 'debit', 'daily')
  end

  def ach_credit_daily_limit
    limits.dig('ach', 'credit', 'daily')
  end
  ...
end
```

Reply:
While it's nice intent to remap the Unit structure into something more consumable, it wondering if keeping more closely aligned to the API docs will make this easier to maintain (and fewer edge cases/inconsistencies in the implementation).

You could still remap things to be friendlier in the API consumer (for us, that means in our app) rather than the base library